packages/tupaia-web-server/src/app/createApp.ts

```diff
@@ -94,6 +94,11 @@ export async function createApp(db: TupaiaDatabase = new TupaiaDatabase()) {
     )
     .use('downloadFiles', forwardRequest(CENTRAL_API_URL, { authHandlerProvider }))
     // Forward everything else to webConfigApi
+    .use('landingPage/:landingPageUrl', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
+    .use('login/oneTimeLogin', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
+    .use('resendEmail', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
+    .use('resetPassword', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
+    .use('signup', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
     .use('*', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }));
```

Comment:
Not sure if this still belongs?

Reply:
I think it shouldn't be a catchall if we are sure that all the endpoints used are covered